readme.md

```diff
@@ -0,0 +1,23 @@
+# Welcome 
+
+Welcome to the Courtgrabber Documentation public repository. These files have been formatted according to the MkDocs theme and are best viewed on an MkDocs site. We have not yet found a permanent hosting service, but we hope to make the move soon. In the meantime, you can view the files by downloading MkDocs, cloning this repo, and running the built-in MkDocs dev server to view the site locally.
+
+## Installing MkDocs
+
+Run the following command from your command line to install MkDocs:
+
+`pip install mkdocs`
```

Comment:
Nit: enclosing commands in single backticks is generally done when the command is inline with other non-monospaced text, like a regular sentence.  For commands (or code in general) that's in a standalone block, it's moe the norm to enclose it in triple backticks.
```suggestion
```
pip install mkdocs
```
```

Reply:
Right. I forgot. Thanks.